app/forms/support_interface/application_forms/update_offered_course_option_form.rb

```diff
@@ -13,10 +13,7 @@ def save(application_choice)
 
         return false unless valid?
 
-        application_choice.update!(
-          current_course_option_id: course_option_id,
-          audit_comment: audit_comment,
-        )
+        application_choice.update_course_option!(course_option, audit_comment: audit_comment)
```

Comment:
We might have another instance we need to change in support: `AddCourseChoiceAfterSubmission`

Reply:
Thanks, amended.